x/stakeibc/keeper/msg_server_restore_interchain_account_test.go

```diff
@@ -48,8 +47,7 @@ func (s *KeeperTestSuite) SetupRestoreInterchainAccount(createDelegationICAChann
 	var channelID, portID string
 	if createDelegationICAChannel {
 		owner := "GAIA.DELEGATION"
-		channelID = s.CreateICAChannel(owner)
-		portID = icatypes.PortPrefix + owner
+		channelID, portID = s.CreateICAChannel(owner)
```

Comment:
Is the addition of the portID return arg to `CreateICAChannel` for the sole purpose of simplifying this logic? Not against it necessarily, just trying to understand

Reply:
yeah exactly. It's called in a lot of setup functions and I've found that every time I go to create a channel in a test, I wind up referencing/copying-pasting those lines haha

I think we should also start adding stakeibc specific test helpers to keeper_test.go (e.g. `CreateDelegationICAChannel`, but that's a ticket for another day!